mmpose/models/detectors/top_down.py

```diff
@@ -153,25 +154,28 @@ def forward_train(self, img, target, target_weight, img_metas, **kwargs):
             # target_weight: [batch_size, num_joints, 1]
             losses['mse_loss'] = self.loss(output, target, target_weight)
 
-        if isinstance(output, list):
-            if target.dim() == 5 and target_weight.dim() == 4:
-                _, avg_acc, _ = pose_pck_accuracy(
-                    output[-1].detach().cpu().numpy(),
-                    target[:, -1, ...].detach().cpu().numpy(),
-                    target_weight[:, -1,
-                                  ...].detach().cpu().numpy().squeeze(-1) > 0)
-                # Only use the last output for prediction
+        if not self.target_type == 'CombinedTarget':
+            if isinstance(output, list):
+                if target.dim() == 5 and target_weight.dim() == 4:
+                    _, avg_acc, _ = pose_pck_accuracy(
+                        output[-1].detach().cpu().numpy(),
+                        target[:, -1, ...].detach().cpu().numpy(),
+                        target_weight[:, -1,
+                                      ...].detach().cpu().numpy().squeeze(-1) >
+                        0)
+                    # Only use the last output for prediction
+                else:
+                    _, avg_acc, _ = pose_pck_accuracy(
+                        output[-1].detach().cpu().numpy(),
+                        target.detach().cpu().numpy(),
+                        target_weight.detach().cpu().numpy().squeeze(-1) > 0)
             else:
                 _, avg_acc, _ = pose_pck_accuracy(
-                    output[-1].detach().cpu().numpy(),
+                    output.detach().cpu().numpy(),
                     target.detach().cpu().numpy(),
                     target_weight.detach().cpu().numpy().squeeze(-1) > 0)
         else:
-            _, avg_acc, _ = pose_pck_accuracy(
-                output.detach().cpu().numpy(),
-                target.detach().cpu().numpy(),
-                target_weight.detach().cpu().numpy().squeeze(-1) > 0)
-
+            avg_acc = 0.0
         losses['acc_pose'] = float(avg_acc)
 
```

Comment:
if self.target_type == 'GaussianHeatMap':
            if isinstance(output, list):
                if target.dim() == 5 and target_weight.dim() == 4:
                    _, avg_acc, _ = pose_pck_accuracy(
                        output[-1].detach().cpu().numpy(),
                        target[:, -1, ...].detach().cpu().numpy(),
                        target_weight[:, -1,
                                      ...].detach().cpu().numpy().squeeze(-1) >
                        0)
                    # Only use the last output for prediction
                else:
                    _, avg_acc, _ = pose_pck_accuracy(
                        output[-1].detach().cpu().numpy(),
                        target.detach().cpu().numpy(),
                        target_weight.detach().cpu().numpy().squeeze(-1) > 0)
            else:
                _, avg_acc, _ = pose_pck_accuracy(
                    output.detach().cpu().numpy(),
                    target.detach().cpu().numpy(),
                    target_weight.detach().cpu().numpy().squeeze(-1) > 0)
            losses['acc_pose'] = float(avg_acc)

If target_type is equal to'CombinedTarget',  we do not need to output acc_pose

Reply:
done